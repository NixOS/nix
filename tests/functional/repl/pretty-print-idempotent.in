:l pretty-print-idempotent.nix
oneDeep
oneDeep
twoDeep
twoDeep
oneDeepList
oneDeepList
twoDeepList
twoDeepList
