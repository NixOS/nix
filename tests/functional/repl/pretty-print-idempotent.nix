{
  oneDeep = {
    homepage = "https://" + "example.com";
  };
  twoDeep = {
    layerOne = {
      homepage = "https://" + "example.com";
    };
  };

  oneDeepList = [
    ("https://" + "example.com")
  ];
  twoDeepList = [
    [
      ("https://" + "example.com")
    ]
  ];
}
