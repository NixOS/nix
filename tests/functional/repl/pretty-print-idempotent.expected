Nix <nix version>
Type :? for help.
Added <number omitted> variables.

{ homepage = "https://example.com"; }

{ homepage = "https://example.com"; }

{
  layerOne = { ... };
}

{
  layerOne = { ... };
}

[ "https://example.com" ]

[ "https://example.com" ]

[
  [ ... ]
]

[
  [ ... ]
]


